sympy/vector/coordsysrect.py

```diff
@@ -170,6 +180,16 @@ def __str__(self, printer=None):
     def __iter__(self):
         return iter([self.i, self.j, self.k])
 
+    def __getattr__(self, name):
```

Comment:
Oh, sorry I misread. I think we should not focus on such minor details in the API, I would rather focus on the algorithms and how to store data. The convenient ways to deal with the API can be thought of later.